Dockerfile_browser_tools.dockerfile

```diff
@@ -10,10 +10,7 @@ RUN sudo add-apt-repository --yes ppa:malteworld/ppa
 RUN sudo apt install pdftk --allow-unauthenticated
 
 # Install Puppeteer via Yarn
-RUN yarn add puppeteer@21.9.0
-
-# Install Chrome using Puppeteer command
-RUN npx puppeteer browsers install chrome@121
+RUN yarn add puppeteer@22.1.0
 
 # Tell Puppeteer to skip installing Chromium. We'll be using the installed package.
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
```

Comment:
I wonder if this needs to be above line 13 to have any effect.

Reply:
also it may not be  needed anymore if we are relying on puppetteer installing chromium

Reply:
Thanks, have now removed. Will test if this works after sorting out the other Docker file.